gui/README.md

```diff
@@ -10,7 +10,7 @@ See the [Getting Started](../docs/getting-started.md) to set up your Cassy confi
 
 Once you have started Cassy, go to the root folder and run the following command to start the HTTP/1.1 server.
 ```
-./entrypoint -dev
+./entrypoint -mode=dev
```

Comment:
I think `prod` should be used for normal users.
There might need a explanation for both dev and prod.